hayeon/baekjoon/그리디/2839-설탕-배달.js

```diff
@@ -0,0 +1,39 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const input = [];
+
+rl.on("line", function (line) {
+  input.push(line);
+}).on("close", function () {
+  // 풀이
+  let kg = +input[0];
+  let count = 0;
+
+  while (true) {
+    if (kg % 5 === 0) {
+      count += Math.floor(kg / 5);
+
+      break;
+    } else if (kg < 0) {
+      count = -1;
+
+      break;
+    }
+
+    kg -= 3;
+    count++;
+  }
+
+  console.log(count);
```

Comment:
```suggestion
  let kg = +input[0];
  let count = 0;

  while (true) {
    if (kg % 5 === 0) {
      count += kg / 5;
      break;
    } 
    if (kg < 0) {
      count = -1;
      break;
    }

    kg -= 3;
    count++;
  }

  console.log(count);
```
- 풀이잘봤씁니다 🙇‍♂️
- 어차피 5로 나눴을 때 나머지가 존재하지 않을 때 5로 나눈 몫을 count로 누적시켜주는 것이기 때문에 Math.floor는 필요없는 코드입니다!!

Reply:
헉 그릏군여!! 맞네요....와우 감사합니다!!!😮👍🏻